### ISSUE TITLE
nodetool getsstable failed; error: getSSTablesForKey(java.lang.String, boolean)

### ISSUE THREAD

**Issue by bhalevy** [scylladb/scylla-jmx#78]
As seen in e.g. [scylla-3.1/job/dtest-release/118/testReport/nodetool_additional_test/TestNodetool/get_sstable_test](https://jenkins.scylladb.com/view/scylla-3.1/job/scylla-3.1/job/dtest-release/118/testReport/nodetool_additional_test/TestNodetool/get_sstable_test/):
```
Error Message
Nodetool command '/jenkins/workspace/scylla-3.1/dtest-release/scylla/resources/cassandra/bin/nodetool -h localhost -p 7149 getsstables ks1 tbl1 4' failed; exit status: 2; stderr: error: getSSTablesForKey(java.lang.String, boolean)
-- StackTrace --
java.lang.NoSuchMethodException: getSSTablesForKey(java.lang.String, boolean)
	at com.sun.jmx.mbeanserver.PerInterface.noSuchMethod(PerInterface.java:169)
	at com.sun.jmx.mbeanserver.PerInterface.invoke(PerInterface.java:135)
	at com.sun.jmx.mbeanserver.MBeanSupport.invoke(MBeanSupport.java:252)
	at com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.invoke(DefaultMBeanServerInterceptor.java:819)
	at com.sun.jmx.mbeanserver.JmxMBeanServer.invoke(JmxMBeanServer.java:801)
	at com.scylladb.jmx.utils.APIMBeanServer.invoke(APIMBeanServer.java:168)
	at javax.management.remote.rmi.RMIConnectionImpl.doOperation(RMIConnectionImpl.java:1468)
	at javax.management.remote.rmi.RMIConnectionImpl.access$300(RMIConnectionImpl.java:76)
	at javax.management.remote.rmi.RMIConnectionImpl$PrivilegedOperation.run(RMIConnectionImpl.java:1309)
	at javax.management.remote.rmi.RMIConnectionImpl.doPrivilegedOperation(RMIConnectionImpl.java:1401)
	at javax.management.remote.rmi.RMIConnectionImpl.invoke(RMIConnectionImpl.java:829)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:357)
	at sun.rmi.transport.Transport$1.run(Transport.java:200)
	at sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:573)
	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:834)
	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:688)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:687)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

This issue is opened for backporting purposes.
The issue is actually fixed by pull #69 (ff0723abc6c2dd23313f93cd5ec396ffd731d743)

**Comment by bhalevy** [comment]
Cc @penberg @amnonh

**Comment by bhalevy** [comment]
Backported and fixed with a8b9849480d8318faad073091d15c491e6d419d8
